src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ function(add_npm_command)
   cmake_parse_arguments(NC "${options}" "${single_kw}" "${multi_kw}" ${ARGN})
   string(REPLACE ";" " " command "${NC_COMMAND}")
   if(NC_NODEENV)
-    string(REGEX REPLACE "^(npm .*)$" ". ${mgr-dashboard-nodeenv}/bin/activate && \\1 && deactivate" command ${command})
+    string(REGEX REPLACE "^(npm .*)$" ". ${mgr-dashboard-node-env}/bin/activate && \\1 && deactivate" command ${command})
```

Comment:
why?

Reply:
@tchaikov 
Because there was a target with the same name as a file path.
Which is confusing?

Reply:
i don't think this change is in the scope of this PR.

Reply:
@tchaikov 
It's like fixing spaces and layout, whilest being there.